examples/with-viem-and-passkeys/src/pages/api/subOrg.ts

```diff
@@ -0,0 +1,90 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { TurnkeyApiTypes, TurnkeyClient } from "@turnkey/http";
+import { createActivityPoller } from "@turnkey/http/dist/async";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+
+type TAttestation = TurnkeyApiTypes["v1Attestation"];
+
+type CreateSubOrgRequest = {
+  subOrgName: string;
+  challenge: string;
+  attestation: TAttestation;
+};
+
+type CreateSubOrgResponse = {
+  subOrgId: string;
+};
+
+type ErrorMessage = {
+  message: string;
+};
+
+export default async function createUser(
+  req: NextApiRequest,
+  res: NextApiResponse<CreateSubOrgResponse | ErrorMessage>
+) {
+  const createSubOrgRequest = req.body as CreateSubOrgRequest;
+
+  try {
+    const turnkeyClient = new TurnkeyClient(
+      { baseUrl: process.env.NEXT_PUBLIC_TURNKEY_API_BASE_URL! },
+      new ApiKeyStamper({
+        apiPublicKey: process.env.API_PUBLIC_KEY!,
+        apiPrivateKey: process.env.API_PRIVATE_KEY!,
+      })
+    );
+
+    const activityPoller = createActivityPoller({
+      client: turnkeyClient,
+      requestFn: turnkeyClient.createSubOrganization.bind(turnkeyClient),
+    });
```

Comment:
This interface is kind of awkward, especially the `.bind` part. If you don't do `.bind`, the "this" value inside of `turnkeyClient` refers to the object with `client` and `requestFn` rather than the `turnkeyClient` itself, and it messes up the resolution of `this.request`.

Is there a cleaner option? 🤔 

@MaxBlaushild @andrewkmin

Reply:
This is the issue that arrow functions have been invented to fix, although javascripters have turned their backs on them for some reason. 

You can either re-declare all of the functions as arrow functions or use a pattern like:

```
requestFn: (...args) => turnkeyClient.createSubOrganization(args)
```

I think both are fine/better than binding

Reply:
Resolved with arrow functions!